### PR TITLE
Update Platform.Hashing to 0.4.0

### PR DESCRIPTION
**Pull request by nassipkali**
Specify library name and version:  **platform.hashing/0.4.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): update platform.hashing to 0.4.0

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/platform.hashing/all/conanfile.py [hunk 1]
```diff
@@ -47,10 +47,6 @@ def _compilers_minimum_version(self):
     def _minimum_cpp_standard(self):
         return 20
 
-    def requirements(self):
-        if Version(self.version) >= "0.3.0":
-            self.requires("platform.delegates/0.2.7")
```
Comment (prince-chrismc) [inline recipes/platform.hashing/all/conanfile.py:52]: Why is this removed? 🤔 it was a added because of a bug AFAIK

**Review by prince-chrismc — COMMENTED** [review]
Looks goof, just one linter comment if you can sneak that in

File: recipes/platform.hashing/all/conanfile.py (hunk 1 above)
Comment (nassipkali) [reply recipes/platform.hashing/all/conanfile.py:52]: one of our developers suggested that this package would be useful in development, but in the end it was not used when the new version was released

**Review by prince-chrismc — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by Konard** [comment]
Any news on 3rd approval?

**Review by Konard — APPROVED** [review]

**Comment by prince-chrismc** [comment]
> Any news on 3rd approval?

The information in the GitHub UI doesnt make what rules we have for the automation https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#rule-of-2-reviews for details

----

The PR has not be run because OP has not been granted access, you need request access from #4 - if you already have then you need to wait for the test round of new users to be added 😄

**Review by ericLemanissier — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
